lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -1675,14 +1675,8 @@ def setUp(self):
         # Create course, seed permissions roles, and create team
         self.course = CourseFactory.create()
         seed_permissions_roles(self.course.id)
-        verified_coursemode = CourseModeFactory.create(
-            course_id=self.course.id,
-            mode_slug=CourseMode.VERIFIED
-        )
-        audit_coursemode = CourseModeFactory.create(
-            course_id=self.course.id,
-            mode_slug=CourseMode.AUDIT
-        )
+        verified_coursemode = CourseMode.VERIFIED
```

Comment:
When the history table was added to CourseEnrollments, tests started failing because CourseEnrollment is looking for a slug, not a CourseMode object. Once I fixed these the test started failing, see below for more info.